src/Core/src/Handlers/ScrollView/ScrollViewHandler.iOS.cs

```diff
@@ -33,6 +33,8 @@ public override bool NeedsContainer
 			}
 		}
 
+		internal ScrollToRequest? PendingScrollToRequest { get; private set; }
```

Comment:
Does this need to be cleared in any other place? `DisconnectHandler()`?

Reply:
Yeah, this should be cleared in DisconnectHandler just in case the handler gets re-used.

Reply:
Cleared in DisconnectHandler.